app/controllers/capitals_controller.rb

```diff
@@ -1,12 +1,28 @@
 class CapitalsController < ApplicationController
+  resource_description do
+    short 'Add capital details to an assessment'
+    formats ['json']
+    description <<-END_OF_TEXT
+    == Description
+      Adds details of an applicant's capital assets to an assessment.
+
+      There are two types of assets:
+
+        - liquid capital items: These are bank accounts.  The description should hold the Bank name and account
+          number, and the value should hold the lowest balance during that calculation period (i.e. the
+          month leading up to the submission date)
+
+        - non-liquid capital items:  These are other capital assets which are not immediately realisable as cash, such
+          as stocks and shares, interest in a trust, valuable items, etc.  Do not include property or vehicles which
+          are added through separate endpoints.
+    END_OF_TEXT
+  end
   api :POST, 'assessments/:assessment_id/capitals', 'Create capital declarations'
   formats ['json']
   param :assessment_id, :uuid, required: true
-  param :liquid_capital, Hash, desc: 'Defines liquid capital assets' do
-    param :bank_accounts, Array, required: true, desc: 'Collection of bank accounts' do
-      param :name, String, required: true, desc: 'An identifying name for this bank account'
-      param :lowest_balance, :currency, required: true, desc: 'The lowest balance on this bank account during the calculation period'
-    end
+  param :liquid_capital, Array, desc: 'Collection of bank accounts' do
+    param :description, String, required: true, desc: 'An identifying name for this bank account'
```

Comment:
I like the fact that you removed the extra hierarchy. But wouldn't `bank_accounts` more sense than `liquid_capital` here?
Could it be something other than a bank account?